### DISCUSSION TITLE
Give DT the same multiplier for mania as other game modes

### DISCUSSION THREAD

**Discussion by Hydria** [ppy/osu#21780] [Ideas]
Happy holidays everyone

I'm aware that this is more or less probably on your radar already but this is more of a call to potentially get this in sooner rather than later, and now we have recent score/SR changes in place I don't think there's anything technically holding back adding this feature in right now:

- PP is now completely based off of acc and not score so having a multiplier wont directly affect PP
- Bonus PP based on score was also removed
- Brings all game modes more in-line with each other
- Spotlights team want this in place to incentivise higher level players to try it more often without them getting bored grinding lower SR maps (since we now allow freemods)
- Also allows DT scores to stand out more in the leaderboards

Can leave this open for a bit to see if anyone has any opposing arguments to this but hopefully it can be a nice edition in the soon future, even if it ruins the perfect million max score that's been in place for so long.

**Comment by smoogipoo** [comment]
This eventually happen, but note that the DT multiplier(s) are still up in the air at the moment so it'll likely happen in tandem with that.
